README.md

```diff
@@ -27,25 +117,148 @@ mtx.myTranspose();
         [2, 4, 6]
        ]
 ```
+Not sure that this is what you wanted -
+```js
+Array.prototype.myTranspose = function() {
+  let result = [[],[]];
+  for (let i = 0; i<this.length; i++) {
+  result[0].push(this[i][0]);
+  result[1].push(this[i][1]);
+  console.log(result)
+}
+return result;
+}
+
+console.log(mtx.myTranspose());
+```
 
 7. Use `myMap` to build a new array where every element is incremented by 10.
-8. Use `myMap` to build a new array where every non-string element is converted to an empty string. 
+```js
+let arr = [1,2,3,4,5];
+let newArr = arr.myMap(el => {return el + 10;})
+console.log(newArr)
+```
+8. Use `myMap` to build a new array where every non-string element is converted to an empty string.
+```js
```

Comment:
This questions was asking for input to be a mixed array and for all the non-strings to be converted to "". [1, "hello"] // => ["", "hello"]